templates/user_detail.html

```diff
@@ -0,0 +1,93 @@
+                  </p>
+                  <div class="ml-2 flex flex-shrink-0">
+                    {% if project.status == "retired" %}
+                      {% pill sr_only="Status:" variant="warning" text=project.get_status_display %}
+                    {% elif project.status == "ongoing" %}
+                      {% pill sr_only="Status:" variant="primary" text=project.get_status_display %}
+                    {% elif project.status == "ongoing-and-linked" %}
+                      {% pill sr_only="Status:" variant="primary" text=project.get_status_display %}
+                    {% elif project.status == "postponed" %}
+                      {% pill sr_only="Status:" variant="danger" text=project.get_status_display %}
+                    {% elif project.status == "completed-and-linked" %}
+                      {% pill sr_only="Status:" variant="success" text=project.get_status_display %}
+                    {% elif project.status == "completed-and-awaiting" %}
+                      {% pill sr_only="Status:" variant="success" text=project.get_status_display %}
+                    {% endif %}
+                  </div>
+                </div>
+                <div class="mt-2 sm:flex sm:justify-between">
+                  <div class="sm:flex">
+                    <p class="flex items-center text-sm text-slate-600">
+                      {% icon_user_group_outline class="mr-1.5 h-5 w-5 flex-shrink-0 text-slate-400" %}
+                      {{ project.member_count }}
+                      user{{ project.member_count|pluralize }}
+                    </p>
+                    <p class="flex items-center text-sm text-slate-600 mt-2 sm:mt-0 sm:ml-6">
+                      {% icon_rectangle_stack_outline class="mr-1.5 h-5 w-5 flex-shrink-0 text-slate-400" %}
+                      {{ project.workspace_count }}
+                      workspace{{ project.workspace_count|pluralize }}
+                    </p>
+                  </div>
+                  <div class="mt-2 flex items-center text-sm text-slate-600 sm:mt-0">
+                    {% icon_calendar_outline class="mr-1.5 h-5 w-5 flex-shrink-0 text-slate-400" %}
+                    <time datetime="{{ project.created_at|date:"Y-m-d H:i:sO" }}">
+                      <span class="sr-only">Project created:</span>
+                      {{ project.created_at|date:"d M Y" }}
+                    </time>
+                  </div>
+                </div>
+              </div>
+            </a>
+          </li>
+        {% endfor %}
+      {% /list_group %}
+    {% /card %}
+  {% else %}
+    {% #card %}
+      <div class="text-center p-8">
+        {% icon_folder_outline class="mx-auto h-12 w-12 text-slate-400" %}
+        <h2 class="mt-2 text-lg font-semibold text-slate-900">
+          No projects
+        </h2>
+        <p class="mt-2 text-base text-slate-600">
+          This user has been attached to any projects yet
+        </p>
+      </div>
+    {% /card %}
```

Comment:
This can be replaced by the empty list item.

```suggestion
    {% #card %}
      {% #list_group %}
        {% list_group_empty icon=True title="No projects" description="This user has been attached to any projects yet" %}
      {% /list_group %}
    {% /card %}
```